t/43-scheduling-and-worker-scalability.t

```diff
@@ -0,0 +1,190 @@
+#!/usr/bin/env perl
+# Copyright (C) 2020 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+
+use strict;
+use warnings;
+
+use FindBin;
+use lib "$FindBin::Bin/lib";
+use OpenQA::Scheduler::Model::Jobs;
+use OpenQA::Test::Database;
+use OpenQA::Jobs::Constants;
+use OpenQA::Test::Utils
+  qw(create_user_for_workers create_webapi setup_share_dir create_websocket_server stop_service setup_fullstack_temp_dir);
```

Comment:
I suggest to use multiple qw(). I know you don't like it but the additional line which is still overlength isn't pretty either.